src/App.jsx

```diff
@@ -1,28 +1,29 @@
-import logo from "./logo.svg";
-import "./App.css";
-import { BrowserRouter, Route, Routes, route } from "react-router-dom";
-import { Provider } from "react-redux";
-import store from "./redux/mainStore";
-import { useDispatch } from "react-redux";
-import { useEffect } from "react";
-import { fetchData } from "./redux/data/dataSlice";
+// import logo from './logo.svg';
```

Comment:
- [ ] please remove the commented code. this will keep your work clean

Reply:
done